src/test/java/org/pesmypetcare/webservice/dao/WeekTrainingDaoTest.java

```diff
@@ -0,0 +1,380 @@
+package org.pesmypetcare.webservice.dao;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.QuerySnapshot;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.firestore.DocumentSnapshot;import org.junit.jupiter.api.BeforeAll;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [';' is not followed by whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)